src/EditorFeatures/Test/CodeGeneration/ExpressionGenerationTests.cs

```diff
@@ -125,21 +125,21 @@ public void TestDoubleLiteralExpression1()
                 vb: "0.5R");
         }
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/26586")]
+        [Fact]
         public void TestAddExpression1()
         {
-            Test(
+            TestWithSimplifier(
                 f => f.AddExpression(
                     f.LiteralExpression(1),
                     f.LiteralExpression(2)),
                 cs: "1 + 2",
                 vb: "1 + 2");
```

Comment:
my preference is that TestWithSimplifier would also run the normal test.  So this should actually take 4 strings.  cs, csSimple, vb, vbSimple. 

After all, the point of this file is to test the generator not the simplifier, so we want to ensure we're always at *least* testing the generator output.

Reply:
Then it doesn't make so much sense to have two different test methods. I'll merge it all into the Test method. I initially separated them because some tests appeared to not result in valid syntax when used as a local declaration. Can pass null to csSimple/vbSimple if that's the case then.